src/pages/Users/helper.tsx

```diff
@@ -0,0 +1,25 @@
+export const FittingDataUserPage = (usersDataResponse) => {
```

Comment:
this is not a constructor! so why for the love of God do you name it from the capital letter?

Reply:
and let's stick to `map` instead of `fitting`

Reply:
function Renamed and repaired to map() insteed forEach()